app/js/controllers/main.js

```diff
@@ -50,22 +48,34 @@ export default class MainController extends Controller {
       return;
     }
 
-    this.view.customizer.setRootNode(document.documentElement);
+    // Insert the customizer view into the app's document
     document.body.appendChild(this.view.el);
 
-    var padding = this.view.el.querySelector('fxos-customizer').offsetHeight;
-    this._styleEl = document.createElement('style');
-    this._styleEl.textContent = `
-    html, body {
-      overflow: initial !important;
-    }
+    // And display it. This has to be in a setTimeout or the CSS animation
+    // won't work correctly since we just added it above.
+    setTimeout(() => { this.view.show(); });
```

Comment:
I looked it up. Can't leave out the parens if there are no args, but I can leave out the braces if the body is just a single expression!  Simplest case is for single-arg functions when you can leave out the parens x=>x+1 for example.  But in this case I need ()=>this.view.show().  Good to get rid of braces, though.